hack/jsonschema/main.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+import os
+import functools
+
+import json
+
+
+def relative_path(filepath):
+    return os.path.join(os.path.dirname(__file__), filepath)
+
+def remove_intxx_formats(path, v):
+    if len(path) > 0 and path[-1] == 'format' and (v == 'int32' or v == 'int64'):
+        return None
+    return v
+
+
+def parameter_string_or_number(path, v):
+    if path == 'definitions/io.argoproj.workflow.v1alpha1.Parameter/properties/default/type'.split('/') \
```

Comment:
this is odd - this should be string type only - numbers are tolerated in the parser - but not considered actually valid

Reply:
Fixed. Wasn't sure which way to go, picked the wrong way.